crates/typst-library/src/foundations/args.rs

```diff
@@ -9,13 +9,14 @@ use crate::foundations::{
     cast, func, repr, scope, ty, Array, Dict, FromValue, IntoValue, Repr, Str, Value,
 };
 
-/// Captured arguments to a function.
+/// 関数に渡される引数。
```

Comment:
capturedはテクニカルタームなので
```suggestion
/// 関数にキャプチャされた引数。
```
を提案します。